documentation/onboarding.md

```diff
@@ -0,0 +1,152 @@
+
+* Manage meetings and huddles
+* Manage teams
+* Integrate external calendars
+* Manage multiple calendars in one space
+
+To create a new account, navigate to the desktop application and select to open. On the sign-in screen that appears, select **[!UICONTROL Create an account]**.
+
+![The Agenda desktop app Sign-In screen, with the Create and account button highlighted.](./images/onboarding/create-an-account.png)
+
+Enter your name and email address, then choose a password and confirm that same password before selecting **[!UICONTROL Create account]**.
+
+![The Create an account screen, with required fields and the Create a account button highlighted.](./images/onboarding/complete-create-account.png)
+
+If successful, you are given confirmation that the account is created. Select **[UICONTROL Close]** and you are returned to the **Sign In** screen.
+
+![The successful account creation dialog, with the Close button highlighted.](./images/onboarding/account-creation-successful.png)
+
+## Integrate an external calendar
+
+You can integrate external calendars into Agenda, allowing you to import and sync events and manage these from a single space.
+
+To integrate an external calendar, log into the Agenda application and select **[!UICONTROL Sources]** from the top navigation, then select **[!UICONTROL Add Calendar sources]** from the left navigation.
+
+![The Sources tab, with Calendar sources selected.](./images/onboarding/calendar-sources.png)
+
+A list of available sources is displayed. Find the service provider for your account and select **[!UICONTROL Connect]**.
+
+![Calendar sources, with a calendar source highlighted.](./images/onboarding/select-source.png)
+
+Enter your account's email address and password within the login dialog, then select **[UICONTROL Connect]**.
+
+![The calendar source login dialog box, with the Connect button highlighted.](./images/onboarding/calendar-source-login.png)
+
+A list of calendars associated with the account appear. Select the check box beside the name of each calendar that you would like to import, then select **[!UICONTROL Import calendars]**.
+
+![The calendars available for import, with the calendar check boxes and Import calendars button highlighted.](./images/onboarding/select-calendars.png)
+
+A message confirms that the calendars have been imported successfully. Select **[UICONTROL Close]** and you are returned to the **[!UICONTROL Sources]** tab.
+
+![The successful calendar import dialog, with the Close button highlighted.](./images/onboarding/import-calendars-successful.png)
+
+Your calendars will now appear in the **[!UICONTROL Calendar]** tab, where they can be used to add reminders, schedule meetings, and so on.
+
+## Next steps
+
+By reading this document, you have successfully downloaded the Agenda desktop client to your machine, created an account, and integrated an external calendar. See the following documents for more information: 
+
+* [Agenda overview](./overview.md).
```

Comment:
Period at the end can be removed since this is a simple list item and not a full sentence